src/woo_publications/metadata/update_informatie_category.py

```diff
@@ -0,0 +1,82 @@
+from io import StringIO
+from pathlib import Path
+
+from django.core.management import call_command
+
+import requests
+from glom import PathAccessError, T, glom
+
+from .constants import InformationCategoryOrigins
+from .models import InformationCategory
+
+WAARDENLIJST_URL = (
+    "https://repository.officiele-overheidspublicaties.nl/waardelijsten/scw_woo_informatiecategorieen/3/json/scw_woo_informatiecategorieen_3.json"
+)
+
+SPEC = {
+    "naam": T["http://www.w3.org/2004/02/skos/core#prefLabel"][0]["@value"],
+    "naam_meervoud": T[
+        "https://identifier.overheid.nl/tooi/def/ont/prefLabelVoorGroepen"
+    ][0]["@value"],
+    "definitie": T["http://www.w3.org/2004/02/skos/core#definition"][0]["@value"],
+    "order": T["http://www.w3.org/ns/shacl#order"][0]["@value"],
+}
+
+
+class InformatieCategoryWaardenlijstError(Exception):
+    def __init__(self, message: str):
+        self.message = message
+        super().__init__(message)
+
+
+def update_informatie_category(file_path: Path):
+    try:
+        response = requests.get(WAARDENLIJST_URL)
+    except requests.RequestException as err:
+        raise InformatieCategoryWaardenlijstError(
+            "Could not retrieve the value list data."
+        ) from err
+
+    try:
+        response.raise_for_status()
+    except requests.RequestException as err:
+        raise InformatieCategoryWaardenlijstError(
+            f"Got an unexpected response status code when retrieving the value list data: {response.status_code}."
+        ) from err
+
+    data = response.json()
+    if not data:
+        raise InformatieCategoryWaardenlijstError(
+            "Received empty data from value list."
+        )
+
+    for waardenlijst in data:
+        # filter out all ids that aren't waardenlijsten
+        if (
+            not waardenlijst.get("@type")[0]
```

Comment:
`.get(...)` can return `None`, so you'd get a type error here because `None[0]` doesn't work.

```suggestion
            not waardenlijst["@type"][0]
```